recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+     ]
+    }
+   ],
+   "source": [
+    "t = Trainee(features=features)\n",
+    "\n",
+    "action_features = ['target']\n",
+    "context_features = features.get_names(without=action_features)\n",
+    "\n",
+    "t.train(df)\n",
+    "\n",
+    "t.analyze(context_features=context_features, action_features=action_features)\n",
+    "\n",
+    "t.react_into_trainee(action_feature=action_features[0], residuals=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Prediction Stats\n",
+    "\n",
+    "Howso provides a variety of prediction stats based on the variable type, whether it is continuous or nominal. These prediction stats gives insight into the predictive performance of the Trainee as well as insights into the data itself. \n",
+    "\n",
+    "The [bias_mitigation.ipynb](https://github.com/howsoai/howso-engine-recipes/blob/main/recipes/2-Workflows/bias_mitigation/bias_mitigation.ipynb) recipe highlights looking for bias in the data through the analysis of individual features. This recipe demonstrates how insights like overall data bias can be explored."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Global Prediction Stats\n",
+    "\n",
+    "\n",
+    "Global prediction stats provides an overall view on how accurate the Trainee is. This can be a great introductory look into the usability of the data and Trainee. Most machine learning models are evaluated on similiar global metrics.\n",
+    "\n",
+    "For more information on the Global statistics, see the [Global vs Local](https://docs.howso.com/user_guide/concepts/global_vs_local.html) documentation.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "accuracy          0.829000\n",
+       "mcc               0.530553\n",
+       "precision         0.737685\n",
+       "recall            0.796072\n",
+       "mae               0.239578\n",
+       "spearman_coeff         NaN\n",
+       "rmse                   NaN\n",
+       "r2                     NaN\n",
+       "Name: target, dtype: float64"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "global_stats = t.get_prediction_stats()['target']\n",
+    "global_stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Local Prediction Stats\n",
+    "\n",
+    "In addition to global prediction stats, Howso has the unique ability to provide local prediction stats. These prediction stats are more finely tuned to the exact cases in which you are interested in. This can be extremely valuable for workflows such as data exploration and it provides more nuanced performance metrics. The more variance there is in the performance of dataset from region to region, the more powerful this ability is."
```

Comment:
Please define a bit more here local vs. global. One sentence is sufficient, but we need to provide context to the users.

Reply:
Also, could you please explain here what we are doing. For example, "Above, we should the overall model statistics, but now let's look at the performance for our individual test case."

Reply:
And then, a quick explanation. For example, "our specific test case has an accuracy of 70%". And, could we format that answer better, instead of just returning 0.7?